seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kafka/KafkaIT.java

```diff
@@ -292,6 +293,30 @@ public void testSourceKafkaStartConfig(TestContainer container)
         testKafkaGroupOffsetsToConsole(container);
     }
 
+    @TestTemplate
+    @DisabledOnContainer(TestContainerId.SPARK_2_4)
+    public void testFakeSourceToKafkaAvroFormat(TestContainer container)
+            throws IOException, InterruptedException {
+        Container.ExecResult execResult =
+                container.executeJob("/avro/fake_source_to_kafka_avro_format.conf");
+        Assertions.assertEquals(0, execResult.getExitCode(), execResult.getStderr());
+    }
+
+    @TestTemplate
+    @DisabledOnContainer(TestContainerId.SPARK_2_4)
+    public void testKafkaAvroToConsole(TestContainer container)
+            throws IOException, InterruptedException {
+        DefaultSeaTunnelRowSerializer serializer =
+                DefaultSeaTunnelRowSerializer.create(
+                        "test_avro_topic",
+                        SEATUNNEL_ROW_TYPE,
+                        MessageFormat.AVRO,
+                        DEFAULT_FIELD_DELIMITER);
+        generateTestData(row -> serializer.serializeRow(row), 0, 100);
```

Comment:
You can do this by adding e2e uniformly to All formats are integrated in this
`seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kafka/KafkaFormatIT.java`

Reply:
hi @zhilinli123, i use a method in `KafkaIT` and also `KafKaIT` has other format e2e test like json, text. so i don't want move to `KafkaFormatIT`.